packages/perseus/src/widgets/interactive-graphs/graphs/linear-system.tsx

```diff
@@ -56,7 +121,82 @@ const LinearSystemGraph = (props: LinearSystemGraphProps) => {
                     color="var(--movable-line-stroke-color)"
                 />
             ))}
-            ;
-        </>
+            {linesAriaInfo.map(
+                ({
+                    pointsDescriptionId,
+                    interceptDescriptionId,
+                    slopeDescriptionId,
+                    pointsDescription,
+                    interceptDescription,
+                    slopeDescription,
+                }) => (
+                    <>
+                        <g
+                            key={pointsDescriptionId}
+                            id={pointsDescriptionId}
+                            style={a11y.srOnly}
+                        >
+                            {pointsDescription}
+                        </g>
+                        <g
+                            key={interceptDescriptionId}
+                            id={interceptDescriptionId}
+                            style={a11y.srOnly}
+                        >
+                            {interceptDescription}
+                        </g>
+                        <g
+                            key={slopeDescriptionId}
+                            id={slopeDescriptionId}
+                            style={a11y.srOnly}
+                        >
+                            {slopeDescription}
+                        </g>
+                    </>
+                ),
+            )}
+        </g>
     );
 };
+
+function LinearSystemGraphDescription({
+    state,
+}: {
+    state: LinearSystemGraphState;
+}) {
```

Comment:
a nit since I still don't quite understand the purpose of the destructuring that's happening here
```suggestion
function LinearSystemGraphDescription(
    state: LinearSystemGraphState) {
```

Reply:
This is a component! The `{state}` is the props. It needs to be like that because it gets called like `<LinearSystemGraphDescription state={state} />`.

Reply:
Oh I seeee
There's an alternative way to declare functional components that might make the distinction between a FC and a regular function more apparent to the layman like me 
```ts
type LinearSystemGraphDescriptionProps = {
 state: LinearSystemGraphState
}

const LinearSystemGraphDescription: React.FC<LinearSystemGraphDescriptionProps> =({state}) => {
```